docs/content/en/main/concepts/investigations/edge-multi-cluster.md

```diff
@@ -0,0 +1,82 @@
+   edge cluster locations, even if they are offline at times, is a key requirement of the edge multi-cluster use case.
+
+## Overlap of roles, responsibilities, and security
+
+Today's smartphone is not only the property of the user who purchased it. The reality is that application developers (app store),
+ corporate entities (vpn profiles), and the smartphone manufacturer have roles and responsibilities on these devices. Edge cluster
+  locations are no different in this respect. A scalable Edge multi-cluster solution can thrive and advance if there is support for
+   community contribution powered by integration and proper security features. All of these considerations must work across an evolving
+    hierarchy that combines small and large control plane API services.
+
+## Hierarchy and Scale are core considerations
+
+KCP has introduced the concept of a minimal API server which can be used to expand the number of places workloads can be deployed
+  by reducing the size of the control plane to a single binary. Furthermore, KCP's Transparent Multi-cluster (TMC) has made it possible to define
+  and deploy workloads centrally and then fan them out to specific edge locations. We would 
+   like to leverage different topologies (for instance: cascading multiple instances of KCP and Kubernetes cluster) to arrive at a 
+    distributed system capable of reaching any targeted edge cluster with/without edge devices attached. Topology flexibility 
+     will require a scalable database to hold the objects referenced by each of their associated API servers. The default database for 
+      KCP and Kubernetes is etcd. We will investigate alternatives to etcd distribution and replacement databases to increase 
+       scalability without sacrificing performance and resilience and considering factors which could negatively impact cost.
+
+## Areas of investigation
+
+1. Define the high level user Edge multicluster use case
+2. Explore characteristics of most common Kube workload objects that could allow them to be placed in edge cluster locations
+3. Desired placement expression​: Need a way for one center object to express large number of desired copies​
+4. Scheduling/syncing interface​: Need something that scales to large number of destinations​
+5. Rollout control​: Client needs programmatic control of rollout, possibly including domain-specific logic​
+6. Customization: Need a way for one pattern in the center to express how to customize for all the desired destinations​
+7. Status from many destinations​: Center clients may need a way to access status from individual edge copies
+8. Status summarization​: Client needs a way to say how statuses from edge copies are processed/reduced along the way from edge to center​.
+9. Seek consensus in user communities on whether the interfaces are practical
+10. Formalize parts of the prototype into project(s) drawing on the elements above if successful!
```

Comment:
Not clear

Reply:
I replaced with this "10. Formalize parts of the prototype into a solution that represents the requirements of edge multicloud use cases."